doc/en/user/source/services/wcs/install.rst

```diff
@@ -0,0 +1,28 @@
+.. _wcs_install:
+
+Installing the WCS extension
```

Comment:
This should be reworded a bit, it seems any WCS needs to be installed as an extension, instead, regarding this PR, it's just WCS 1.0 that's in the extension (and eventually WCS 1.1 will join it).

WCS 2.0 instead will stay in core, so for that one, no installation is needed.